app/src/main/res/layout/beer_layout.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Beer"
```

Comment:
Extract the string here and place it in strings.xml. You could name it as "beer_text" in the strings.xml resource and reference it using @strings/beer_text